### DISCUSSION TITLE
[New] Le maillage interne est pertinent

### DISCUSSION THREAD

**Discussion by Ferreole** [cnumr/best-practices#97] [Bonnes pratiques] [answered]
Bonne pratique #5110   
--
3, non   prioritaire | 1, MEO facile | 2, impact moyen
Gain :    Temps d’appropriation de l’internaute   Durabilité du contenu | Le maillage interne consiste à favoriser la navigation   à l’intérieur du site.   Son objectif :    faciliter la l’accès à l’information pour l'internaute

**Comment by ACTLEM** [comment, marked as answer]
Suite à la visio du 12/11, je me permets de mettre cette discussion en résolu.
